templates/documentation/datasheet-documentation.html

```diff
@@ -0,0 +1,243 @@
+{% extends '_base.html' %}
+{% load static %}
+{% block title %}Datasheet Documentation{% endblock %}
+
+{% block content %}
+<div class=" font-sans text-center">
+    <div class="pt-12">
+        <p class="text-4xl font-bold leading-7 text-purple-600 pb-12">Datasheet Documentation</p>
+    </div>
+    <div class="flex flex-wrap justify-center text-left pb-6">
+        <div class="text-lg w-3/4 bg-white shadow-lg rounded-2xl md:w-3/4">
+            <div class="container w-full flex flex-wrap mx-auto px-2">
+                <div class="w-full p-8 lg:mt-0 text-black leading-normal bg-white">
+                    <!--Title-->
+                    <div>
+                        <h1 class="break-normal text-black pt-6 pb-2 text-5xl">Overview</h1>
+                        <hr class="border-b border-grey-light">
+                    </div>
+                    <p class="py-6">
+                        Admin will be able to upload CSV files pertaining to 3 categories: Courses, Students, or
+                        Preferences. Uploading a preset CSV will prevent admin from having to manually input respective
+                        data one by one. If Admin does not have any CSV files to upload, please refer to the Database
+                        documentation for next steps [<a class="hover:underline text-blue-500" href="{% url 'documentation:database-documentation' %}">here</a>]
```

Comment:
The "here" link does not work on some of the pages

Reply:
Also, some of the doc links don't work

Reply:
some of the docs where not commited. will commit them with the js